pkg/proxy/config/config.go

```diff
@@ -107,6 +107,8 @@ type EndpointsConfig struct {
 
 // NewEndpointsConfig creates a new EndpointsConfig.
 func NewEndpointsConfig(endpointsInformer coreinformers.EndpointsInformer, resyncPeriod time.Duration) *EndpointsConfig {
+	klog.Infof("creating new endpoints config with resync period of %v", resyncPeriod)
```

Comment:
Log lines usually start with capital letter. Also wondering why this log message is needed, is it to easily determine if we're watching Endpoints or EndpointSlice?

Reply:
The reason i actually added this was because of the generally confusing state of affairs when it comes to kube-proxy config.  We dont print out the entire configuration, and so if someone plumbs a custom value in, its super helpful to know what that value was :)

Reply:
This isn't actually "config" though, this is the endpoints watcher/handler logic named EndpointsConfig.

Reply:
hmmm  are you suggesting we shouldnt log it , or that its incorrect to call it a config, or both ?   resyncPeriod is definetly a configuration parameter (  https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L201 ) at least, in the sense that it is something that varies and is sent in by the user and configured from a configMap ?

Reply:
resyncPeriod is a configuration option passed into the EndpointsConfig, but EndpointsConfig does not actually parse or read the configuration for it, that is done in a previous step by kube-proxy when CLI args are read. 

I do think the name `EndpointsConfig` is a misleading name but I don't think it's worthwhile changing in this PR.

Reply:
ok, so @andrewsykim all i really care about is that we convey the kube-proxy input arguments **somehow** .  this is one of many such args.  Maybe we can do that in a follow on PR somehow ? i.e. just dump all configuration params that are being used on startup ?

Reply:
Other components do dump all flags set on start up, but I think it's only at a certain verbosity level. Recommend checking what kubelet does and do similar? Would prefer a follow-up PR for that

Reply:
I think this is best done via the keep above so... just to expedite this I’ll  move this effort there.....